### PR TITLE
Updated Splunk SDK

### PR DESCRIPTION
**Pull request by dhruvilbhatt-crest**

### CODE REVIEW

**Review by dsommerville-illumio — CHANGES_REQUESTED** [review]
Thanks, looks good. Can you update `app.conf` to bump the build/launcher versions and add an `[id]` stanza?

I think all other appinspect warnings can be safely ignored.

**Comment by dhruvilbhatt-crest** [comment]
@dsommerville-illumio Bumped the version to 4.0.0. Since we are maintaining the version in the launcher stanza, the ID stanza is not required.

**Comment by dsommerville-illumio** [comment]
@dhruvilbhatt-crest - this is not a major version release. Please stick to standard [semantic versioning](https://semver.org/) - this is a backwards-compatible change with no deprecations or additional functionality, so it should just be a patch version bump to 3.2.3.

As for the `[id]` stanza, I'd prefer to duplicate the values to avoid the pre-cert warnings.

File: TA-Illumio/default/app.conf [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # Splunk app configuration file
 
 [install]
-build = 20170419
+build = 1
```
Comment (dsommerville-illumio) [inline TA-Illumio/default/app.conf]: This should conform to the previous build format, which seems to be YYYYMMDD

**Review by dsommerville-illumio — APPROVED** [review]
Thanks, looks good.

**Pull request merged**